Fix(curriculum): external RMS link in CIP (#46389)
freeCodeCamp/freeCodeCamp: curriculum/challenges/english/10-coding-interview-prep/rosetta-code/averages-root-mean-square.md

# --description--

> Compute the Root mean square of the numbers 1 through 10 inclusive.

Reviewer: ```suggestion
Compute the Root Mean Square (RMS) of the numbers 1 through 10 inclusive.
```
I think its standard to define abbreviations like this in line before using them..

> The <abbr title="Root mean square">RMS</abbr> is calculated as the mean of the squares of the numbers, square-rooted:

Reviewer 2: ```suggestion
The <abbr title="Root mean square">RMS</abbr> is calculated by taking the mean of the squares of the numbers, and then square-rooting that number:
```

> The <abbr title="Root mean square">RMS</abbr> is calculated by taking the mean of the squares of the numbers, and then square-rooting that number:

Reviewer: maybe?
```suggestion
The <abbr title="Root mean square">RMS</abbr> is calculated by taking the square root of the mean of the squares of the numbers, given by the equation:
```
